securedrop/manage.py

```diff
@@ -22,13 +22,9 @@
 # We need to import config in each function because we're running the tests
 # directly, so it's important to set the environment correctly, depending on
 # development or testing, before importing config.
-#
-# TODO: do we need to store *_PIDFILE in the application config? It seems like
-# an implementation detail that is specifc to this management script.
-
 os.environ['SECUREDROP_ENV'] = 'dev'
 
-WORKER_PIDFILE = "/tmp/test_rqworker.pid"
+from config import WORKER_PIDFILE
```

Comment:
I had concerns about reusing the PID file as well, but decided it was alright to re-use on the `development` VM. I'll test with a separate PID file specifically for tests and see if that resolves the problem, but I may @ you for refactoring the cleanup defs in the test harness earlier than 0.4.